### PR TITLE
libxslt: fix manpage + remove second distfile

### PR DESCRIPTION
**Pull request by meator**
The second distfile seems superfluous, so this PR removes it. It also fixes `xsltproc(1)`.
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/libxslt/template [hunk 1]
```diff
@@ -28,6 +26,12 @@ post_configure() {
 	fi
 }
 
+post_build() {
+	# xsltproc manpage depends on xsltproc, this remakes it after
+	# libxslt is build
+	make -C doc XSLTPROC=${wrksrc}/xsltproc/xsltproc xsltproc.1
```
Comment (ericonr) [inline srcpkgs/libxslt/template:32]: Any idea why this works for cross?

Comment (meator) [reply srcpkgs/libxslt/template:32]: _That's a really good question._

Comment (meator) [reply srcpkgs/libxslt/template]: I didn't think of that. It (of course) doesn't work for cross, it just ignores errors while generating the manpage. I have included a premade manpage for use in the cross compiled version.

**Comment by meator** [comment]
Making `xsltproc` depend on `xsltproc`? Interesting. Thanks!

**Comment by meator** [comment]
@sgn Why did you leave the second distfile in?

**Comment by sgn** [comment]
> @sgn Why did you leave the second distfile in?

I forget, I don't apply your changes, only leave your name there.

**Pull request closed without merging**